scenes/test_number.txt

```diff
@@ -25,6 +25,20 @@
 *gosub test_finish result_expected cslib_ret
 
 
+*comment :::::: MAX_STAT tests ::::::
+*gosub test_start "MAX_STAT #1"
+*temp result_expected "test_stat_strength"
+*gosub_scene cslib_number max_stat "test_stat_strength" "test_stat_charisma" "test_stat_wisdom"
```

Comment:
This might sound odd, but can we actually \*set the test_stat values here?
That will:
- Allow future tests to reuse the test_stat variables (startup is gonna look horrible if every test has its own vars)
- Make it immediately obvious to a reader why test_stat_strength is the highest

Reply:
Done